README.md

```diff
@@ -0,0 +1,195 @@
+# OOP Ruby Project
+
+## Getting Started
+
+This repository is a practice with ruby language.
+
+- Use [ruby.rb](./ruby.rb) to decode your morse message
+
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+
+<!-- TABLE OF CONTENTS -->
```

Comment:
- [ ] Kindly make sure to clean up the README file from all the comments so it looks clean and easy to navigate on a text editor